Extension/src/LanguageServer/configurations.ts

```diff
@@ -1092,11 +1093,13 @@ export class CppProperties {
             }
 
             if (configuration.compileCommands) {
-                configuration.compileCommands = this.resolvePath(configuration.compileCommands);
-                if (!this.compileCommandsFileWatcherFallbackTime.has(configuration.compileCommands)) {
-                    // Start tracking the fallback time for a new path.
-                    this.compileCommandsFileWatcherFallbackTime.set(configuration.compileCommands, new Date());
-                }
+                configuration.compileCommands = configuration.compileCommands?.map((path: string) => this.resolvePath(path));
```

Comment:
minor: I don't believe you need to use `?.` here or on the line below because the existence of `configuration.compileCommands` was tested before entering this code block.

Reply:
fixed